extensions/blocks/slideshow/slideshow.js

```diff
@@ -116,6 +116,12 @@ class Slideshow extends Component {
 
 	render() {
 		const { autoplay, className, delay, effect, images } = this.props;
+
+		// If user has not selected any images, don't print any markup.
+		if ( ! images.length ) {
```

Comment:
This could probably happen further up the function, too, before to understructure the `props` object. Also, not sure but maybe it worths check if `images` is `undefined`.

```es6
// check if images has items...
if ( ! this.props?.images?.length ) {
  return null;
}

// Now, time to understructure the object!
const { autoplay, className, delay, effect, images } = this.props;
```

Reply:
Sure! Thanks !